### ISSUE TITLE
resource "oci_load_balancer"  ssh_cipher_suite unusable

### ISSUE THREAD

**Issue by eugsim1** [terraform-providers/terraform-provider-oci#1263]
Error: Unsupported block type

```
  on modules/loadbalancer/loadbalancer.tf line 16, in resource "oci_load_balancer" "load_balancer":
  16:     ssl_cipher_suites {
```

Blocks of type "ssl_cipher_suites" are not expected here.

[```
oracle@localhost loadbalancer-4]$ terraform -v
Terraform v0.13.5
+ provider registry.terraform.io/hashicorp/local v2.0.0
+ provider registry.terraform.io/hashicorp/null v3.0.0
+ provider registry.terraform.io/hashicorp/oci v4.7.0
+ provider registry.terraform.io/hashicorp/template v2.2.0
+ provider registry.terraform.io/hashicorp/tls v3.0.0
```

the following uncommented code is giving the above issue , 
when the code is commend out there is no error, the doc is wrong :

  ```
/*
    ssl_cipher_suites {
        name = "oci-wider-compatible-ssl-cipher-suite-v1" ##var.load_balancer_ssl_cipher_suites_name
    } 
   	*/
   
   from the docs :	
   ssl_cipher_suites {
        #Required
        name = var.load_balancer_ssl_cipher_suites_name
    }
```

my code with comments to work 

```
resource "oci_load_balancer" "load_balancer" {
  shape          = var.loadbalancer.shape ###  
  compartment_id = var.loadbalancer.compartment_ocid
  subnet_ids     = var.loadbalancer.subnet_ids
  display_name   = var.loadbalancer.display_name  ##"load_balancer"
  ip_mode        = var.loadbalancer.ip_mode       ##"IPV4"
  is_private     = var.loadbalancer.private       ###"true"
  shape_details {
    maximum_bandwidth_in_mbps = var.loadbalancer.maximum_bandwidth_in_mbps 
    minimum_bandwidth_in_mbps = var.loadbalancer.minimum_bandwidth_in_mbps 
  }
  /*
    ssl_cipher_suites {
        name = "oci-wider-compatible-ssl-cipher-suite-v1" ##var.load_balancer_ssl_cipher_suites_name
    } 
   	*/
}

```

Please update the docs to the correct content
thanks

**Comment by srishtipmishra** [comment]
@eugsim1 - Thanks for bringing this to notice. We'll update our doc to rectify this.

**Comment by srishtipmishra** [comment]
Fixed in Provider 4.9 https://github.com/terraform-providers/terraform-provider-oci/blob/master/CHANGELOG.md
Closing this issue.
